### DISCUSSION TITLE
Syncing non-includable files (f.e. .docx, .xlsx)

### DISCUSSION THREAD

**Discussion by Gewerd-Strauss** [Vinzent03/obsidian-git#291] [Q&A] [answered]
Hello,

Recently I began working with more `.docx`- and a couple of `.xlsx`-files during my work, which are put into various subfolders of my vault, which relate to various notes and are mainly launched from there.

Since then, I noticed that those kind of files are not synced to GH - which initially confused me.. Then I realised that the plugin probably only considered files which can be included and are displayed in obsidian's internal FE get synced; but then again ObsidianSync itself _does_ sync `.docx` etc, so obsidian must have access to the files on some level at least.

However, it would be useful if the plugin would sync the vault in its entirety, regardless of the file type - so that formal documents arising from notes can be synced and backed up alongside them. 
I am aware that this might architecturally not be possible, and will likely depend entirely on what obsidian exposes to the plugin's environment. Furthermore, due to the announcement of 0.16 and the corresponding large amount of changes it is told to contain this could become possible or impossible after 0.16 releases; it therefore might be better to hold off until it is released before attempting a solution.

Would that be possible and could it be done (maybe post 0.16-launch)?

Thank you.  
Sincerely,  
~Gw

**Comment by Vinzent03** [comment, marked as answer]
I don't know why your docx files are not committed. I'm pretty sure you have a .gitignore somewhere. 0.16 is purely visual so this doesn't affect this plugin's functionality. You could try running `git status` in the terminal.

**Reply by Gewerd-Strauss** [reply]
> I'm pretty sure you have a .gitignore somewhere.
Yeah, and according to local logs I apparently overwrote it 2 months back with another repo's instance accidentally, which contained docx and xlsx files. 

Thank you.
